File: engine/engine-model/src/main/scala/tools/packet/mtsoperation/ShowMTSCash.scala
package tools.packet.mtsoperation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowMTSCash(private var _maplePoint: Int, private var _nxPrepaid: Int) extends PacketInput {
  def maplePoint: Int = _maplePoint

  def nxPrepaid: Int = _nxPrepaid

  override def opcode(): SendOpcode = SendOpcode.MTS_OPERATION2
}

File: engine/engine-model/src/main/java/tools/packet/mtsoperation/ShowMTSCash.java
package tools.packet.mtsoperation;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowMTSCash(Integer maplePoint, Integer nxPrepaid) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MTS_OPERATION;
   }
}